### ISSUE TITLE
Replacing help

### ISSUE THREAD

**Issue by pwalczysko** [ome/omero-guides#2]
When trying to move all from to this repo, it seems that the missing chapters in the guides here (which are present in help) are

- make annotations workflows more dense ? (step-by-step with screenshots how to add tags,...)
- Managing user settings
- User roles
- Group types
- Summary of roles and permissions in groups
- Viewing data by groups
- Moving data between groups
- Changing your default group
- Sharing data with users you are not in a group with (maybe dont want to push this)
- Managing groups as group owner
- Managing groups as an OMERO Administrator
- Search - detailed wildcard search examples and KVP searches, advanced search
- Publishing with OMERO
- Using OMERO.webtagging
- URLs for Linking to data
- Managing an OMERO Group (add a user, remove a user, moving data (again ?))
- mtools workflows
- virtual microscope
- FlimFit
- Troubleshooting OMERO (how to submit an error, sysadmin troubleshoot page, memory mngmt, where is insight log)
- Contact OME

cc @jburel @will-moore

**Comment by jburel** [comment]
Thanks for the list. I don't think we need to migrate things like flimfit or mtools

**Comment by jburel** [comment]
Managing users and groups could be our next chapter

**Comment by jburel** [comment]
starting page https://www.openmicroscopy.org/omero/features/analyze/

**Comment by jburel** [comment]
meeting 18/06: attending @joshmoore  @pwalczysko @mtbc and @jburel:
* Decision redirect help to guide. Prepare a sheet indicating which page to redirect too
* Page without corresponding page yet will be redirected to "under construction"
* Page like flimfit, mtools, sls -> redirected to a pdf/doc
- [ ] activate google analytics on readthedocs. check the status with subproject
- [ ] review link www.openmicroscopy

**Comment by jburel** [comment]
for background https://docs.google.com/spreadsheets/d/1pgU1Xxx1yx59xEg8r5tl6hJALtDjPOMZ3_08iLGzJgM/edit#gid=0

**Comment by jburel** [comment]
https://www.openmicroscopy.org/omero/features/index.html
* roiFolder: insight only we will need to find a link
* histogram: add screenshot in viewer guide.

**Comment by jburel** [comment]
https://www.openmicroscopy.org/omero/features/publish/
Movie creation is not a feature that we really promote.

**Comment by jburel** [comment]
https://www.openmicroscopy.org/omero/features/view/
* HCS data. We probably need to add specific screenshot in web section about browsing a plate

**Comment by jburel** [comment]
www.openmicroscopy.org/omero/features/analyze/
* Wnd-charm should be removed. Pointing to an archived repo

**Comment by pwalczysko** [comment]
> HCS data. We probably need to add specific screenshot in web section about browsing a plate

@jburel Web section of what ? Of guides (i.e. not existing guide yet) ? Or a web section of the www webpage ?

**Comment by pwalczysko** [comment]
> roiFolder: insight only we will need to find a link

Do you want me to start on a guide ? We would probably need to think about how to formulate such guide, as the feature is a bit unused

**Comment by jburel** [comment]
I don't think we should embark on writing a guide at this stage. It is not present in the web and the way we interact with  the ROI structure will probably change with the zarr work.
We will probably need to revisit www.openmicroscopy.org

**Comment by jburel** [comment]
For HCS: I was thinking that adding screenshot of HCS data under data management will be a good start or we have add a new page alongside data management so it is clearer than OMERO support HCS data
